packages/webpack/mixins/fast-build/mixin.core.js

```diff
@@ -0,0 +1,45 @@
+const { Mixin } = require('hops-bootstrap');
+
+function include(rules, path) {
+  const shouldInclude =
+    // allow everything outside of node_modules
+    !path.includes('node_modules') ||
+    // allow all hops packages
+    /node_modules[/\\]hops/.test(path) ||
+    // check if any other rule matches
+    rules.some((rule) => rule.test(path));
+
+  return shouldInclude;
+}
+
+class HopsFastBuildMixin extends Mixin {
+  configureBuild(webpackConfig, { jsLoaderConfig }, target) {
+    const enabled =
+      this.options.fastBuild || (this.options.fastDev && target === 'develop');
+
+    if (!enabled) {
+      return;
+    }
+
+    process.env.__HOPS_FAST_BUILD__ = 'true';
+
+    const {
+      experimental: { fastBuild: { includeRules = [] } = {} } = {},
```

Comment:
here they're named `includeRules`, down in the `preset.js` they're named `buildIncludeRules`. however, I think I'd prefer `babelIncludeRules`, because there's a lot to in- & exclude when dealing with webpack. :D